### PR TITLE
[MTO] Updated installation process for MTO

### PR DESCRIPTION
**Pull request by hussnain612**
Updated the installation process of MTO from OperatorHub

### CODE REVIEW

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-55-91f8c815`

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-55-0b70badd`

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-55-7fff6d02`

File: docs/content/sre/tenant-operator/installation.md [hunk 1]
```diff
@@ -248,7 +247,7 @@ helm install tenant-operator stakater/tenant-operator --namespace stakater-tenan
 --set image.repository=stakaterdockerhubpullroot/tenant-operator \
 --set imagePullSecrets[0].name=stakater-docker-secret \
 --set resources.limits.cpu=600m \
```
Comment (abdulhaseeb2) [inline docs/content/sre/tenant-operator/installation.md]: Increase CPU to 1000m

File: docs/content/sre/tenant-operator/installation.md [hunk 2]
```diff
@@ -293,20 +292,20 @@ spec:
...
     image:
       repository: stakaterdockerhubpullroot/tenant-operator
-      tag:  v0.5.2
+      tag:  v0.7.0
       pullPolicy: IfNotPresent
     imagePullSecrets:
     - name: stakater-docker-secret
     resources:
       limits:
         cpu: 600m
```
Comment (abdulhaseeb2) [inline docs/content/sre/tenant-operator/installation.md]: Increase CPU to 1000m

**Review by abdulhaseeb2 — CHANGES_REQUESTED** [review]

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-55-db46eb75`

**Review by MuneebAijaz — APPROVED** [review]

**Review by abdulhaseeb2 — APPROVED** [review]

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-55-fa8fd11d`

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.34`

**Pull request merged**